recipes/diffpy-structure/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "diffpy.structure" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "de481329c4dc41ce1538f88fac768c8f677860602caa886fbfb3429bc17ac3ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
- I suggest to use distribution archive from pypi.io as it has the package metadata already built.
- please update to 1.3.5.